src/mcp_mt5/main.py

```
@@ -0,0 +1,1107 @@
+        
+    Returns:
+        List[Dict[str, Any]]: List of ticks.
+    """
+    # Convert datetime to timestamp in milliseconds
+    date_from_timestamp = int(date_from.timestamp() * 1000)
+    date_to_timestamp = int(date_to.timestamp() * 1000)
+    
+    ticks = mt5.copy_ticks_range(symbol, date_from_timestamp, date_to_timestamp, flags)
+    if ticks is None:
+        logger.error(f"Failed to copy ticks for {symbol} in range {date_from} to {date_to}, error code: {mt5.last_error()}")
+        raise ValueError(f"Failed to copy ticks for {symbol} in range {date_from} to {date_to}")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)